R/get_merge_call.R

```diff
@@ -361,7 +361,10 @@ get_anl_relabel_call <- function(columns_source, datasets, anl_name = "ANL") {
         if (rlang::is_empty(column_names)) {
           return(NULL)
         }
-        column_labels <- datasets$get_varlabels(attr(selector, "dataname"), column_names)
+
+        data_used <- data[attr(selector, "dataname")]
+        labels <- formatters::var_labels(data[attr(selector, "dataname")][[1]], fill = FALSE)
```

Comment:
data is a list of reactives
```suggestion
        labels <- formatters::var_labels(data[[attr(selector, "dataname")]](), fill = FALSE)
```